lib/standards/aria-roles.js

```diff
@@ -169,7 +170,7 @@ const ariaRoles = {
     prohibitedAttrs: ['aria-label', 'aria-labelledby']
   },
   dialog: {
-    type: 'widget',
+    type: 'window',
```

Comment:
Probably `structure` again as not a landmark

```suggestion
    type: 'structure',
```

Reply:
Since we said `window` is fine for `alertdialog` I assume it's fine here as well